themes/salix/layouts/partials/head.html

```diff
@@ -5,7 +5,10 @@
 	<title>{{.Site.Title}}</title>
 	<link rel="preconnect" href="https://fonts.gstatic.com">
 	<link
-			href="https://fonts.googleapis.com/css2?family=Alegreya:wght@400;700&display=swap"
+			href="https://fonts.googleapis.com/css2?family=Alegreya:wght@400;700;900&display=swap"
```

Comment:
Might be worth thinking about not using Google Fonts since privacy is part of our values statement. That can be a later PR though and doesn't need to be part of this, since it was already in there. I'm just thinking out loud since I think I added it in the first place. Thinking carefully about our dependencies might be a good general principal to follow.

Reply:
Makes sense, I think we can just download them and send them ourselves. Probably won't be a performance issue with netlify.

I'll do it as a separate pull request like you said and keep this one focused on getting everything looking right.